CHANGELOG.md

```diff
@@ -8,12 +8,15 @@
 ### Fixed
 * Allow backlinks between files. ([#1015](https://github.com/realm/realm-dart/issues/1015))
 * Fix issue with accessing properties after traversing a backlink. ([#1018](https://github.com/realm/realm-dart/issues/1018))
+* Fix a race condition which could result in "operation cancelled" errors being delivered to async open callbacks rather than the actual sync error which caused things to fail (Core upgrade, since the introduction of async open).
```

Comment:
We're not using async open in the dart SDK.

Reply:
Removed